stoneforge/petrophysics/shale_volume.py

```diff
@@ -21,13 +21,17 @@ def gammarayindex(gr, grmin, grmax):
     
     """
 
+    if grmin == grmax:
+        msg = "The value of grmin is equal to the value of grmax. Division by zero encountered"
```

Comment:
msg = "division by zero. The value of grmin is equal to the value of grmax"
would be more similar to the Python default message for this error